crates/bevy_pbr/src/wireframe.rs

```diff
@@ -53,31 +73,180 @@ fn extract_wireframe_config(mut commands: Commands, wireframe_config: Res<Wirefr
     }
 }
 
-fn extract_wireframes(mut commands: Commands, query: Query<Entity, With<Wireframe>>) {
-    for entity in query.iter() {
-        commands.get_or_spawn(entity).insert(Wireframe);
+fn extract_wireframes(mut commands: Commands, query: Query<(Entity, &Wireframe)>) {
+    for (entity, wireframe) in query.iter() {
+        commands.get_or_spawn(entity).insert(wireframe.clone());
+    }
+}
```

Comment:
This can be replaced with `ExtractComponentPlugin::<Wireframe>::extract_visible()` which will only extract the visible wireframes and it will batch-insert them too.